tests/test_vec_extract_dict_obs.py

```diff
@@ -39,6 +39,7 @@ def test_extract_dict_obs():
     env = DictObsVecEnv()
     env = VecExtractDictObs(env, "rgb")
     assert env.reset().shape == (4, 86, 86)
+    env.step(None)
```

Comment:
```suggestion
    env.step([env.action_space.sample() for _ in range(env.num_envs)])
```

Reply:
Thanks for catching that line. It slipped in.
I've updated the test now to verify that ``step()`` correctly modifies obs and terminal_observation.